<ui version="4.0" >
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>MainMenu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="4" >
    <widget class="QFrame" name="titleFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>271</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>37</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="logoLabel" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>200</width>
          <height>37</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="../../resources/graphics/main-menu.qrc" >:/main/neurososd.png</pixmap>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>41</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="3" row="1" column="3" >
    <widget class="QListView" name="listView" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>4</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <property name="isWrapping" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="selectionRectVisible" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="menuiconLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap/>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="4" >
    <widget class="QFrame" name="frame_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>441</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="ximLabel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="../../resources/graphics/main-menu.qrc" >:/main/xim.png</pixmap>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/graphics/main-menu.qrc" />
 </resources>
 <connections/>
</ui>
